### ISSUE TITLE
Troubleshoot effects of the ESUP file-manager on the portal page

### ISSUE THREAD

**Issue by drewwills** [Jasig/uPortal-start#175]
The ESUP file-manager is now on the Welcome tab (quickstart data set) by default.

It's a great addition, but I detect 2 new regressions when the portlet is present:
  - The 'Options' menu does not appear.  There is a JavaScript error relating to the Portlet Print option that doesn't allow the Options menu to finish bootstrapping.  (Odds are goo the Portlet Print feature is no longer working properly.)
  - The user cannot access the right-click menu on the page.

**Comment by jgribonvald** [comment]
In my mind the print option should be removed !
We made the choice to remove it from our feature as it's difficult to maintain it with different kind of portlets and with their content.
On an other way users don't use it a lot or don't understand why they can't have this options on some case. It's a frustration.
Also we should work on mediaqueries to define specifics print css, but it's too much to do !

As the last word I don't know if someone use again this feature (introduced by me for the ESUP context and don't work since uP 4.x) !

**Comment by drewwills** [comment]
I agree the print option should be removed from uPortal.  It never was much used, and if we needed it now (in 2018) I bet there would be a much better way to do it.

**Comment by drewwills** [comment]
@ChristianMurphy & @jgribonvald,

Only this sub-issue remains...

    CSS from the file-manager is changing the appearance of the Calendar portlet (others?)

It would be good to fix it.

For tracking, we could close this ticket and open a new, smaller ticket.

**Comment by jgribonvald** [comment]
@drewwills : to fix the problem we have to watch on jquery-theme and jquery version.

- first change may be that ESUP-FileManager need to follow uPortal jQuery version
- the main problem is on the jquery theme used, there isn't theme context limited to ESUP-Filemanager - I never encountered a such problem so I don't know if this can be solved by configuration. Do you know more about that ?
- Maybe the solution is only on defining the loading of jquery by context, but problems should come from here [on this page](https://github.com/uPortal-contrib/esup-filemanager/blob/master/src/main/webapp/WEB-INF/jsp/view-portlet.jsp)
